frontend/src/components/layout/Navbar.js

```diff
@@ -32,7 +33,7 @@ const Navbar = () => {
               </NavLink>
             </li>
           ))}
-          <NavLink
+           <NavLink
```

Comment:
Underline of this icon is coming as full width. Can you please correct it.

Reply:
@ashishnagpal2498  made the required changes. Plz review it once